sonar-project.properties

```diff
@@ -2,8 +2,6 @@ sonar.projectKey=eslint-plugin-sonarjs
 sonar.projectName=eslint-plugin-sonarjs
 sonar.projectDescription=SonarJS rules for ESLint
 
-# current version in development
-# should be bigger than the one in package.json
```

Comment:
I think it's not up to date anymore as in package.json we have a version currently under development (as we don't publish manually anymore)